packages/components/src/progress-bar/stories/index.story.tsx

```diff
@@ -29,3 +29,37 @@ const Template: StoryFn< typeof ProgressBar > = ( { ...args } ) => {
 
 export const Default: StoryFn< typeof ProgressBar > = Template.bind( {} );
 Default.args = {};
+
+const withCustomWidthCustomCSS = `
+	.custom-progress-bar {
+		width: 100%;
+	}
+`;
+
+/**
+ * A progress bar with a custom overriden width.
```

Comment:
We can remove some redundancy:

```suggestion
 * A progress bar with a custom width.
```